lib/carrierwave/storage/fog.rb

```diff
@@ -145,7 +145,7 @@ def clean_cache!(seconds)
         connection.directories.new(
           :key    => uploader.fog_directory,
           :public => uploader.fog_public
-        ).files.all(:prefix => uploader.cache_dir).each do |file|
+        ).files.all(:prefix => uploader.cache_dir).each_file_this_page do |file|
```

Comment:
Use of `each_file_this_page` disables pagination handling for [fog-aws](https://github.com/fog/fog-aws/blob/master/lib/fog/aws/models/storage/files.rb#L42-L57).
This code is shared across multiple fog providers, you need to make sure that all of them does not break.

Reply:
I couldn't understand how CarrierWave::Storage::Fog#clean_cache! uses pagination handling, sorry.
It would be helpful if you could explain in a little more detail.
Thanks.